src/reducers/profile-view.js

```diff
@@ -635,6 +636,11 @@ export const selectorsForThread = (
         return ProfileData.getCallNodeInfo(stackTable, frameTable, funcTable);
       }
     );
+    const getCallNodeMaxDepth = createSelector(
+      getFilteredThread,
+      getCallNodeInfo,
+      StackTiming.computeCallNodeMaxDepth
```

Comment:
This is more a general thought: should we try to compute this from the full thread instead of the filtered thread ?
I understand that for the stackchart you want the filtered thread: because otherwise it would be too high.
But here I believe it's not so important.

Advantages of using the full data:
* no need to recompute when the filtering changes
* the size of the call chart won't change when the filtering changes => potentially a less important reflow

However I don't think this will give a real difference for the user, but tell me what you think?

Reply:
All good observations. I think for now I'll keep this on the filtered thread so the result is more correct for the data that's presented, plus I think having the unfiltered max depth may be a bit confusing as to why we went that way. My intuition is that it won't have as much affect on performance. That said, I'd be OK with modifying it if we find out otherwise.